pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "scylla"
-requires-python = ">=3.8"
+requires-python = ">=3.12"
```

Comment:
In general we want to support all not-EOL versions of Python. What is the reason for raising the requirement to 3.12?